isoparser/pom.xml

```diff
@@ -1,17 +1,16 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.mp4parser</groupId>
     <artifactId>isoparser</artifactId>
     <packaging>jar</packaging>
     <name>ISO Parser</name>
     <description>A generic parser and writer for all ISO 14496 based files (MP4, Quicktime, DCF, PDCF, ...)
     </description>
     <url>https://github.com/sannies/mp4parser</url>
-    <version>1.9.57-SNAPSHOT</version>
+    <version>1.9.58-SNAPSHOT</version>
```

Comment:
Probably should not be done in this pr. The maintainer is going to have his own workflow.